streem.gemspec

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+src = File.expand_path("src", __dir__)
+$LOAD_PATH.unshift(src) unless $LOAD_PATH.include?(src)
+require "streem/version"
+
+Gem::Specification.new do |spec|
+  spec.name = "streem"
+  spec.version = Streem::VERSION
+  spec.authors = ["Streem Dev"]
+  spec.email = ["dev@streem.pro"]
+
+  spec.summary = "Streem SDK"
+  spec.description = "Streem library to interact with the Streem API and generate Embedded SSO tokens"
+  spec.homepage = "https://streem.pro"
+  spec.required_ruby_version = Gem::Requirement.new(">= 2.4.0")
+
+  spec.metadata["homepage_uri"] = spec.homepage
+  # spec.metadata["source_code_uri"] = "TODO: Put your gem's public repo URL here."
+  # spec.metadata["changelog_uri"] = "TODO: Put your gem's CHANGELOG.md URL here."
```

Comment:
Can you go ahead and configure these, since we are moving this code to be public?  I simple Changelog that just mentions the initial release, as a starting point, is fine.